site/src/components/Head.tsx

```diff
@@ -2,6 +2,7 @@ import { graphql, useStaticQuery } from "gatsby"
 import "normalize.css"
 import * as React from "react"
 import { Helmet } from "react-helmet"
+const favicon = require("../favicon.favicon.png")
```

Comment:
Should this be `../favicon/favicon.png` ?

Reply:
Yes. It is… unclear to me why this didn't break locally.